### ISSUE TITLE
bug: Markdown links not highlighted in correct color

### ISSUE THREAD

**Issue by oSumAtrIX** [ReVanced/revanced-manager#1962]
### Bug description

Links should be accentuated:

![image](https://github.com/ReVanced/revanced-manager/assets/13122796/f5cf0025-d9cf-414b-87e2-8b9c4ec1ccf0)

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The bug is only related to ReVanced Manager

**Comment by Ushie** [comment]
Yep, was waiting on https://github.com/mikepenz/multiplatform-markdown-renderer/issues/77 back then
